frmts/vrt/pixelfunctions.cpp

```diff
@@ -932,14 +1008,34 @@ static CPLErr PowPixelFuncHelper( void **papoSources, int nSources, void *pData,
 
     /* ---- Return success ---- */
     return CE_None;
-}  // PowPixelFuncHelper
+}  // ExpPixelFuncHelper
+
+static CPLErr ExpPixelFunc( void **papoSources, int nSources, void *pData,
+                            int nXSize, int nYSize,
+                            GDALDataType eSrcType, GDALDataType eBufType,
+                            int nPixelSpace, int nLineSpace,
+                            CSLConstList papszArgs )
+{
+    double dfBase = 2.7182818284590452353602874713526624;
+    double dfFact = 1.;
+
+    if ( FetchDoubleArg(papszArgs, "base", &dfBase, &dfBase ) != CE_None )
+        return CE_Failure;
+
+    if ( FetchDoubleArg(papszArgs, "fact", &dfFact, &dfFact ) != CE_None )
```

Comment:
dfFact != 0 should be tested

Reply:
Good point, it is now fixed.

By the way we could also implement ``base ^ ( fact * x )`` instead of ``base ^ ( x / fact )``

Reply:
> By the way we could also implement base ^ ( fact * x )

yes, I was also wondering if that wouldn't be more generic

Reply:
done